src/OmniSharp/MSBuild/MSBuildProjectSystem.cs

```diff
@@ -9,6 +9,7 @@
 using Microsoft.CodeAnalysis.MSBuild;
 using Microsoft.CodeAnalysis.Text;
 using Microsoft.Framework.Logging;
+using OmniSharp.Models;
```

Comment:
IEventEmitter isn't a model. If it's in that interface we should move it to services.